developerportal/templates/molecules/card-featured-primary.html

```diff
@@ -20,7 +20,7 @@
 
 
 <div class="mzp-l-card-full">
-  <section class="mzp-c-card mzp-c-card-x-large mzp-has-aspect-16-9">
+  <section class="mzp-c-card devportal-card-x-large devportal-has-aspect-3-1">
```

Comment:
In a lot of other places(not just this PR), I see `section` elements changed to `div`. That is not the case here. Is there a particular reason to keep this one a `section`?

Reply:
True - the original markup had lots of `section` and I basically am not being consistent enough with replacement. Willdo!